### ISSUE TITLE
Package it onto the PyPi

### ISSUE THREAD

**Issue by ThatXliner** [adamsol/Pyxell#12]
Package it with a setup.py and everything so we can install it on PyPi

**Comment by adamsol** [comment]
I've already had that idea, but failed at creating a package with all the necessary files. It's easy for `.py` source code, but Pyxell needs also some other files, and for some reason I couldn't make it work.

On second thought, I'm not sure whether PyPI is a good place for a programming language. Compilers are usually installed in some other ways. Also, it's only an implementation detail that Pyxell is written in Python. I'm already thinking about rewriting it to C++ to make parsing faster and make it possible to implement an interpreter in a fairly easy way.

**Comment by ThatXliner** [comment]
> I've already had that idea, but failed at creating a package with all the necessary files. It's easy for `.py` source code, but Pyxell needs also some other files, and for some reason I couldn't make it work.

Have you seen the [MANIFEST.in](https://packaging.python.org/guides/using-manifest-in/) file? That's where you can include other files, too.

> 
> 
> 
> On second thought, I'm not sure whether PyPI is a good place for a programming language.

Pip is pre-installed on almost every system. I almost cry when packages say "get this from [homebrew](https://brew.sh)" because my HomeBrew *barely works*. Pip is the easiest way. Or [conda](https://docs.conda.io/en/latest/).

**Comment by adamsol** [comment]
> Have you seen the MANIFEST.in file? That' where you can include other files, too.

Yeah, I tried it, and something still didn't work. It seems to be a strange problem for many people, see e.g. https://stackoverflow.com/questions/1612733/including-non-python-files-with-setup-py. I just gave up after trying several approaches.

> Pip is pre-installed on almost every system. I almost cry when packages say "get this from homebrew" because my HomeBrew barely works. Pip is the easiest way. Or conda.

Maybe I'll have better luck with conda. It also fits better, since it's language-agnostic. However, Pyxell can be easily downloaded from [Github releases](https://github.com/adamsol/Pyxell/releases) and run from source. It's the same way I've recently downloaded [Kotlin](https://github.com/JetBrains/kotlin/releases/tag/v1.4.30).
